notebooks/haystack-qa.ipynb

```diff
@@ -0,0 +1,494 @@
+{
```

Comment:
<p>Great start! The adversarial example was a good addition :D Haystack makes implementing the reader retriever approach quite intuitive. For the source of context, it should have a method to point back to the file, it seems to be a simple string search problem... </p><p>For the PR, I think it has the commits from the file addition PRs as well. I guess if we separate them then this can be merged independently. </p><p>Looking forward to the generative example from haystack as well!</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/foundation-models-for-documentation/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>